dags/config/countries/cameroon.yaml

```diff
@@ -0,0 +1,9 @@
+constants:
+  ISO3: CMR
+  ISO2: CM
+
+hdx_cod:
+  adm:
+    address: cameroon-administrative-boundaries
+    filename: cmr_admbnda_inc_20180104_SHP.zip
+    file_type: shp
```

Comment:
Is this file_type going to be different for other countries?

Reply:
Yes, and I set the default to be `gpkg` in the config file. However I just checked and the HDX API returns a file format -- I will switch to using that and take out this parameter.

Reply:
Now that I've looked into it a bit more, I realized that the above is trickier than I thought. I could query the file type during the initial API call, but then it would need to be passed between tasks. It looks like this is possible with XComs, do you ever use this? Otherwise I could make a second API call in the transformation task. @janvdvegt any thoughts or better ideas?

Reply:
The way Airflow works, it's better to have no API calls during the DAG generation, only inside the actual Operators. Making the call inside the operator would be fine, if we only care about it inside the operator. It's difficult for me to judge that.

Reply:
With the idea I had above, the second API call would also be inside an operator. At any rate, specifying it explicitly works for now, I think I will just create an issue and leave it out of this PR. It's also possible that we may want to only ingest shapefiles in the end, since they seem much more prevalent than geopackages.

Reply:
Created issue #15